en/docs/learn/configuring-password-policies.md

```diff
@@ -4,3 +4,4 @@ WSO2 Identity Server enables configuring password policies to enhance user passw
 
 -	[Password Patterns](../../learn/configuring-password-patterns)
 -	[Password History](../../learn/validating-password-history)
+-   [Password Expiry](../../learn/password-expiry)
```

Comment:
Please check if the last item on the list is aligned properly

Reply:
This appears fine on my IDE. Will change it later if it appears misaligned on the site page.